asv_bench/benchmarks/benchmarks.py

```diff
@@ -407,6 +407,7 @@ def setup(self, shape, ngroups=5, subset="all"):
                 f"Invalid value for 'subset={subset}'. Allowed: {list(self.subset_params.keys())}"
             )
         ncols = subset(shape)
+        ngroups = translator_groupby_ngroups(ngroups, shape)
```

Comment:
could we just inherit `BaseTimeValueCounts` from `BaseTimeGroupBy` to ease the maintainability of `value_counts` benchmarks, since we now don't need to sync 'ngroups' parameter handling changes between these two related classes?

Reply:
I do not want to introduce an additional connection between these two classes, due to the specifics of ASV benchmarks, it is better to create classes as independent of each other as possible.